edgexfoundry/edgex-compose
# fix: Add env override back to C device services

This is require since the C Device services don't implement common config yet.

Note: MQTT bus override will not be generated and if needed have to be added by handle.

Also add work addoned for bacnet command still having old --confdir flag







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
run compose builder `make run ds-bacnet ds-coap no-secty`
Verify the service bootstrap successfully
